bin/yugabyted

```diff
@@ -545,17 +545,29 @@ class ControlScript(object):
                     " Removing...").format(data_dir_files, data_dir))
                 rmcontents(data_dir, exclude_names=[pid_file_name])
 
+            # Add enforced authentication flags on the basis of command-line flag
+            if self.configs.saved_data.get("ysql_enable_auth"):
```

Comment:
we should move this to outside this loop, otherwise we will keep adding to yb_tserver_cmd each time this loop runs. For ex: we could do this in start_processes

Reply:
Yeah, we can add this snippet at 491 just after, where extracting the extra YB tserver gflags from `tserver_flags`.